nextjs-app/components/pages/main/WhatWeBring.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import Button from "@/components/common/Button";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+
+interface GoalCardProps {
+  imageSrc: string;
+  title: string;
+  subTitle: string;
+  description: string;
+}
+
+const GoalCard = (props: GoalCardProps) => {
```

Comment:
```suggestion
const GoalCard = ({ imageSrc, title, subTitle, description }) => {
```
prefer destruct here

Reply:
FYI, you can use React’s FC to define prop types

```tsx
import type { FC } from 'react'

...

const GoalCard: FC<GoalCardProps> = ({
  imageSrc,
  title,
  subTitle,
  description,
}) => {
```